frontend/src/app/modals/add-block-form/add-block-form.component.css

```diff
@@ -152,6 +165,11 @@ td.mat-mdc-cell:first-of-type, td.mat-mdc-footer-cell:first-of-type{
     background: #90CAF9;
 }
 
+::ng-deep .mat-mdc-form-field.wayToAddBlockDropDown .mat-mdc-select-value-text {
```

Comment:
Geht vielleicht ohne ng-deep?

Reply:
Die css Klasse ist schon in styles vorhanden. Möglicherweise geht es deshalb nicht, die Klasse im Component css zu überschreiben.
<img width="243" alt="image" src="https://github.com/adessoSE/Seed-Test/assets/81535134/274737df-31a8-4672-b73c-41fadc131f0d">